cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1442,11 +1453,15 @@ if(tmva)
 
   if(python AND tmva-pymva)
     message(STATUS "Looking for Numpy")
-    find_package(NumPy)
-    if(fail-on-missing AND NOT NUMPY_FOUND)
+    if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
+      find_package(Python COMPONENTS NumPy)
+    else()
+      find_package(NumPy)
+    endif()
+    if(fail-on-missing AND NOT (NUMPY_FOUND OR Python_NumPy_FOUND))
```

Comment:
We can `set(NUMPY_FOUND ${Python_NumPy_FOUND})` after line 1457 and keep changes more succint.